mlflow/server/handlers.py

```diff
@@ -2661,6 +2662,51 @@ def _delete_logged_model_tag(model_id: str, tag_key: str):
     return _wrap_response(DeleteLoggedModelTag.Response())
 
 
+@catch_mlflow_exception
+@_disable_if_artifacts_only
+def _search_logged_models():
+    request_message = _get_request_message(
+        SearchLoggedModels(),
+        schema={
+            "experiment_ids": [
+                _assert_array,
+                _assert_item_type_string,
+                _assert_required,
+            ],
+            "filter": [_assert_string],
+            "max_results": [_assert_intlike],
+            "order_by": [_assert_array],
+            "page_token": [_assert_string],
+        },
+    )
+    models = _get_tracking_store().search_logged_models(
+        # Convert `RepeatedScalarContainer` objects (experiment_ids and order_by) to `list`
+        # to avoid serialization issues
+        experiment_ids=list(request_message.experiment_ids),
+        filter_string=request_message.filter or None,
+        max_results=request_message.max_results or None,
```

Comment:
We should give it a default value instead of None?

Reply:
If `max_results` is unspecified, it defaults to 0 (due to proto). To avoid ambiguity, use None instead of 0.

Reply:
```suggestion
        max_results=request_message.max_results,
```

works if `search_logged_models` treats 0 as `None`.